Source/WebCore/animation/ScrollTimeline.cpp

```diff
@@ -173,4 +174,10 @@ ScrollTimeline::Data ScrollTimeline::computeScrollTimelineData() const
     return { maxScrollOffset, scrollOffset };
 }
 
+std::optional<CSSNumberishTime> ScrollTimeline::currentTime()
+{
+    auto data = computeScrollTimelineData();
+    return CSSNumberishTime(CSSNumericFactory::percent(data.scrollOffset / data.maxScrollOffset));
```

Comment:
We should improve `CSSNumberishTime` to have a constructor that takes in a percentage as input without having to go through a `CSSNumericValue`. This should be reserved for use through bindings.

I expect in most cases we'll want to create the percentage value from a fraction in the 0-1 range, so maybe something like `CSSNumberishTime::percentageFromFraction(double)`?

Note that this should not be blocking, the code as it stands is fine.